packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -113,7 +119,7 @@ abstract class TransitionRoute<T> extends OverlayRoute<T> {
 
   T _result;
 
-  void handleStatusChanged(AnimationStatus status) {
+  void _handleStatusChanged(AnimationStatus status) {
```

Comment:
I guess people could always just reoverride didPush and didReplace and call animation.addStatusListener...